go/cmd/dolt/commands/sqlserver/server_test.go

```diff
@@ -76,7 +76,6 @@ func TestServerBadArgs(t *testing.T) {
 		{"-P", "300"},
 		{"-P", "90000"},
 		{"-u", ""},
-		{"-t", "-1"},
```

Comment:
Had to remove this test case as it now fails earlier. The call to Serve is never made inside of startServer so serverController.WaitForServe will never return a start error.

Reply:
... it still fails.  just not in this way.